test/functions.php

```diff
@@ -0,0 +1,23 @@
+<?php
+require('../functions.php');
+use PHPUnit\Framework\TestCase;
+class collection_appTest extends TestCase
+{
+     public function testSuccessMakePlantEntryTile () {
+        $expected = "<div class='entry_box'><div class='entry science_name'>" . "Querbus robur" . "</div><div class='entry'>" . "English Oak" . "</div><div class='entry'>" . "Tree" . "</div></div>";
+        $this->expectOutputString($expected);
+        makePlantEntryTile(['id' => "1", 'science_name' => "Querbus robur", 'name' => "English Oak", 'type' => "Tree"]);
+    }
+
+    public function testErrorMakePlantEntryTileTooShort () {
```

Comment:
So here I think you are trying to test that if there isn't enough data, it will output 

> `here is not enough data for this entry`

again, im not 100% sure how the expectOutputString works but i think the test you want to use is assertSame($expected,$actual) or assertEqual($expected,$actual).